test/lux/sample-xml-skeleton/population.txt

```diff
@@ -0,0 +1,5 @@
+{string, "populate type string"}.
+{unit8, "16"}.
+{unit32, "32"}.
```

Comment:
uint32 - maybe?
The populate-functionality is really nice.
For instance with typedefs local in the module - you can just add the atom and voila.